components/Arrow/Arrow.tsx

```diff
@@ -1,20 +1,47 @@
-import css from 'styled-jsx/css'
-export default function Arrow(){
+export default function Arrow({arrowOrientation}){
```

Comment:
give it default orientation in props
reasoning:
https://alexkondov.com/tao-of-react
"Prefer assigning default values directly when you’re destructuring the props. It makes it easier to read the code from top to bottom without jumping and keeps the definitions and values together."

```
// 👎 Don't define the default props outside of the function
function Component({ title, tags, subscribed }) {
  return <div>...</div>
}

Component.defaultProps = {
  title: '',
  tags: [],
  subscribed: false,
}

// 👍 Place them in the arguments list
function Component({ title = '', tags = [], subscribed = false }) {
  return <div>...</div>
}
```

Reply:
@danielpopek94